mtp_noms_ops/templates/prisoner_location_admin/location_file_upload.html

```diff
@@ -35,11 +53,6 @@ <h1 class="govuk-heading-xl">{% trans 'Upload prisoner location file' %}</h1>
         </ol>
 
         <div class="govuk-inset-text">
-          <p>
-            {% blocktrans trimmed %}
-              Ensure you use the full report <strong>including the private estate</strong>.
```

Comment:
the user running the report has no control over this, they just need to choose the right report as detailed above